charts/kyverno-policies/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 type: application
 name: kyverno-policies
-version: v2.5.5
-appVersion: v1.7.3
+version: v3.1.1
+appVersion: v1.11.1
```

Comment:
Not sure how to want to manage versions on the main branch.

Reply:
@realshuting I will keep it as it is.